<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageImu</class>
 <widget class="QWidget" name="PageImu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Roll/Pitch/Yaw</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCustomPlot" name="rpyPlot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Accel/Gyro/Mag</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QCustomPlot" name="accelPlot" native="true"/>
         <widget class="QCustomPlot" name="gyroPlot" native="true"/>
         <widget class="QCustomPlot" name="magPlot" native="true"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>3D Plot</string>
      </attribute>
      <layout class="QVBoxLayout" name="layout3d">
       <item>
        <widget class="QCheckBox" name="useYawBox">
         <property name="text">
          <string>Use Yaw (will drift)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
